guides/v2.3/cloud/before/before-workspace-magento-prereqs.md

```diff
@@ -71,20 +72,20 @@ To set `memory_limit`:
 
 Before working with your {{site.data.var.ece}} project, make sure you set the PHP `memory_limit` to at least `1G` for normal use or at least `2G` for debugging.
 
-1.	Find your `php.ini` file using the following command:
+1. Find your `php.ini` file using the following command:
 
-		php --ini
+    php --ini
 
-	Use the value of `Loaded Configuration File`.
-2.	As a user with `root` privileges, open `php.ini` in a text editor.
-3.	Change the value of `memory_limit` to at least `1G` for normal use or at least `2G` for debugging.
-4.	Save your changes to `php.ini` and exit the text editor.
-5.	Restart your web server:
+  Use the value of `Loaded Configuration File`.
```

Comment:
None of these old steps are confirmed accurate. I would drop edits to this old file.

Reply:
I will revert this file.